modules/api/src/main/java/org/apache/ignite/tx/TransactionOptions.java

```diff
@@ -39,10 +39,17 @@ public long timeoutMillis() {
     /**
      * Sets transaction timeout, in milliseconds.
      *
-     * @param timeoutMillis Transaction timeout, in milliseconds.
+     * @param timeoutMillis Transaction timeout, in milliseconds. Cannot be negative; 0 means 'use default timeout'.
+     *     For RO transactions, the default timeout is data availability time configured via ignite.gc.lowWatermark.dataAvailabilityTime
+     *     configuration setting.
+     *     For RW transactions, timeouts are not supported yet. TODO: IGNITE-15936
```

Comment:
Let's extract this todo in a separate comment (outside of the javadoc) for it to be more noticeable (I also wonder if the current approach will work with the TODO detection tool).